libs/deposits/src/lib/deposits-provider.ts

```diff
@@ -41,4 +60,25 @@ export const depositsProvider = makeDataProvider<
     );
     return uniqBy([...incoming, ...(data || [])], 'id');
   },
+  pagination: {
+    getPageInfo,
+    append,
+    first: 1000,
```

Comment:
**question (non-blocking)* *: is 1k a sensible default? It feels high but I'm not very experienced with this api